### PR TITLE
Add authentication and overview endpoints in ref

### PR DESCRIPTION
**Pull request by NolwennP**
2 endpoints are missing from the API reference.
Here they are !

### CODE REVIEW

File: content/akeneo-web-api.yaml [hunk 1]
```diff
@@ -10,7 +10,326 @@ schemes:
...
+          }
+        406:
+          $ref: "#/responses/406Error"
+  /api/oauth/v1/token:
+    post:
+      summary: Get authentication token
+      operationId: "post_autentication_token"
+      tags: 
+        - Authentication
+      description: This endpoint allows you to an authentication token. No need to be authenticated to use this endpoint.
```
Comment (ahocquard) [inline content/akeneo-web-api.yaml]: allows you to get

File: content/akeneo-web-api.yaml [hunk 2]
```diff
@@ -10,7 +10,326 @@ schemes:
...
+      x-no-token: true
+      parameters:
+        - name: Authorization
+          in: header
+          type: string
+          required: true
+          description: Equal to 'Basic xx', where 'xx' is the base 64 encoding of the client id an secret. Find out how to generate them in the <a href="/documentation/security.html#create-an-oauth-client">Create an OAuth client</a> section.
```
Comment (ahocquard) [inline content/akeneo-web-api.yaml]: and secret.

File: content/akeneo-web-api.yaml [hunk 3]
```diff
@@ -10,7 +10,326 @@ schemes:
...
+              grant_type:
+                type: string
+                description: Always equals to "password"
+            example:
+              {
+                "username" : "admin",
+                "password": "admin",
+                "grant_type": "password"
+              }
+
```
Comment (ahocquard) [inline content/akeneo-web-api.yaml]: blank line

File: content/akeneo-web-api.yaml [hunk 4]
```diff
@@ -10,7 +10,326 @@ schemes:
...
+      responses:
+        200:
+          description: Return an authentication token
+          x-no-entity: true
+          schema:
+            type: object
+            properties:
+              access_token:
+                type: string
+                description: Authentication token that should be given in every request to the API
```
Comment (ahocquard) [inline content/akeneo-web-api.yaml]: every authenticated request (token is usless for the get list of endpoint )

File: content/akeneo-web-api.yaml [hunk 5]
```diff
@@ -10,7 +10,326 @@ schemes:
...
+              expires_in:
+                type: integer
+                description: Validity of the token given in milliseconds. 3600ms by default
```
Comment (ahocquard) [inline content/akeneo-web-api.yaml]: in seconds (one hour)

File: content/akeneo-web-api.yaml [hunk 6]
```diff
@@ -10,7 +10,326 @@ schemes:
...
+            properties:
+              username:
+                type: string
+                description: Your PIM username
+              password:
+                type: string
+                description: Your PIM password
+              grant_type:
+                type: string
+                description: Always equals to "password"
```
Comment (ahocquard) [inline content/akeneo-web-api.yaml]: equals or equal ? you used both in this PR.

File: content/akeneo-web-api.yaml [hunk 7]
```diff
@@ -1386,13 +1709,14 @@ paths:
       tags:
         - Media files
       parameters:
-        - name: content-type
+        - name: Content-type
           in: header
           type: string
-          x-value: multipart/form-data
+          description: Equal to 'multipart/form-data', no other value allowed
```
Comment (ahocquard) [inline content/akeneo-web-api.yaml]: Here it's equal.

File: content/akeneo-web-api.yaml [hunk 8]
```diff
@@ -10,7 +10,326 @@ schemes:
...
+                type: string
+                description: Host name
+              authentication:
+                type: object
+                description: Endpoint to get the authentication token
+              routes:
+                type: object
+                description: All the availables endpoints
+          examples: {
+            "host": "http://test-dev-feature-9.akeneo.com",
```
Comment (ahocquard) [inline content/akeneo-web-api.yaml]: `http://akeneo.com` is ok I think

File: content/akeneo-web-api.yaml (hunk 7 above)
Comment (NolwennP) [inline content/akeneo-web-api.yaml]: I chose "Equal". I'm not sure though.

**Pull request merged**